app/models/edition/taggable_organisations.rb

```diff
@@ -11,17 +11,21 @@ module Edition::TaggableOrganisations
   # Only Education based organisations have validation to prevent publishing if
   # they have not been tagged
   def must_be_tagged_to_taxonomy?
-    education_taggable?
+    topic_taggable?
   end
 
   def can_be_tagged_to_taxonomy?
-    education_taggable? || world_taggable?
+    topic_taggable?
+  end
+
+  def can_be_tagged_to_worldwide_taxonomy?
+    world_taggable?
   end
 
 private
 
-  def education_taggable?
-    organisations_in_education_tagging_beta?
+  def topic_taggable?
```

Comment:
Might be confused with specialist topics, or topics (policy areas). Maybe say `topic_taxonomy_taggable?` to disambiguate?